### PR TITLE
feat: new FAQs and docs

### PR DESCRIPTION
**Pull request by insertish**

### CODE REVIEW

File: docs/faq/usage.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+# 6. What can I do with Revolt?
+
+## Brand Guidelines
+
+In general:
+- The Revolt branding is used to represent the platform, revolt.chat.
+- You may use the branding as-is to promote the platform and your community on the platform.
+- You should not use the branding in order to appear as if you are associated with the team.
```
Comment (Zomatree) [inline docs/faq/usage.md]: should probably reword "the team" to be more clear

File: docs/faq/usage.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Please make explicit distinctions between Revolt (the platform) and the Revolt software.
+- The Revolt software is unbranded and only associated by name.
+
+## Guidelines for Private Instances
+
+- You **must** provide correct attribution in line with our software license:
+  - If you are using official images ([GitHub Packages](https://github.com/orgs/revoltchat/packages) / [Docker Hub](https://hub.docker.com/u/revoltchat)), attribution is included.
+  - If you are modifying the software and using it in production, you must publish the changes to the source publicly in line with [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html). (In addition to providing attribution back to the original project.)
+- You are **solely responsible** for whatever happens on your instance, we provide no warranty or liability for what happens on private instances.
```
Comment (Zomatree) [inline docs/faq/usage.md]:
```suggestion
- You are **solely responsible** for whatever happens on your instance, we provide no warranty or liability for what happens on 3rd party instances.

```

File: docs/faq/verification.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+sidebar_position: 7
+---
+
+# 7. How to be verified?
+
+We do not currently offer any means to verify servers or bots.
+
+If you have a compelling case, for example a large open source project, please ask for [@insert in the lounge](https://rvlt.gg/Testers).
```
Comment (Zomatree) [inline docs/faq/verification.md]:
```suggestion
If you have a compelling case (for example, a large open source project), please ask for [@insert in the lounge](https://rvlt.gg/Testers).

```

File: docs/stack.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 3
+---
+
+# Revolt Stack
+
+The diagram below shows how each component interacts with each other.
+
+<img src="/img/diagrams/Revolt Architecture.png" />
```
Comment (Zomatree) [inline docs/stack.md:9]: this is missing quart.

**Review by Zomatree — CHANGES_REQUESTED** [review]

File: docs/faq/monetisation.md [hunk 5]
```diff
@@ -4,14 +4,17 @@ sidebar_position: 4
...
+- Revolt is also incredibly inexpensive to run, as of 7th March 2022, Revolt is currently still running on a bare-minimum cloud VPS.<br/>I previously noted here that we were due an upgrade, but I am since going to revise that to say that the recent optimisation work has brought us below that threshold.
 
 Now you may be asking how will Revolt be funded?
-- We have a variety of monetisation ideas lined up internally, **none of which involve paywalling features**!
-- We currently have £1616 in donations (as of 12th November) from our community, of which only £60 has been spent to run the service (12th November).
+- We have a variety of monetisation ideas lined up internally, with these, it is **not my intention** for us to paywall features and I find it unlikely we would ever do that considering it would contradict what we're trying to achieve.<br/>
+- We have amassed a significant amount of money from donations alone (~ £2k).<br/>This bullet point no longer links to the full donation log as I have moved the spreadsheet to my private cloud so I could manage it with [Grist](https://getgrist.app), it is not possible for me to share this data on demand.
```
Comment (Rexogamer) [inline docs/faq/monetisation.md]: Seems as though `getgrist.app` is down?
```suggestion
- We have amassed a significant amount of money from donations alone (~ £2k).<br/>This bullet point no longer links to the full donation log as I have moved the spreadsheet to my private cloud so I could manage it with [Grist](https://www.getgrist.com/), it is not possible for me to share this data on demand.
```

**Review by Rexogamer — CHANGES_REQUESTED** [review]

File: docs/stack.md (hunk 4 above)
Comment (insertish) [reply docs/stack.md:9]: going to ignore this for now until quark goes live

File: docs/faq/monetisation.md (hunk 5 above)
Comment (insertish) [reply docs/faq/monetisation.md]: Removed additional info here.

**Pull request merged**